tests/core/streams/Stream.spec.ts

```diff
@@ -0,0 +1,128 @@
+import Stream from 'src/core/streams/Stream';
```

Comment:
Would you mind running `yarn lint`? That will update this file to comply with the project's formatting standards. You can then just commit and push the changes.

Reply:
Updated